src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -863,13 +870,23 @@ public static object DeepCopyInner(object original)
             {
```

Comment:
The precedence isn't clear to me. Do we first check for Orleans serializer, and if we find one, we don't even look for the external one? Shouldn't it be the other way around? Or did I misunderstand the code?